pkg/operator/cluster/operator.go

```diff
@@ -139,6 +153,36 @@ func (c *clusterOperator) createStatefulSet(cluster *crv1.MySQLCluster) error {
 	return nil
 }
 
+func (c *clusterOperator) updateServices(cluster *crv1.MySQLCluster) error {
+	serviceInterface := c.kubeClientset.CoreV1().Services(cluster.Namespace)
+
+	err := updateService(cluster, serviceInterface, serviceTemplate)
+	if err != nil {
+		return err
+	}
+
+	err = updateService(cluster, serviceInterface, serviceReadTemplate)
+
+	return err
+}
+
+func updateService(cluster *crv1.MySQLCluster, serviceInterface typedv1.ServiceInterface, template string) error {
+	service, err := serviceForCluster(cluster, template)
+	if err != nil {
+		return err
+	}
+
+	oldService, err := serviceInterface.Get(service.ObjectMeta.Name, metav1.GetOptions{})
+	service.ObjectMeta.ResourceVersion = oldService.ObjectMeta.ResourceVersion
```

Comment:
I guess it's some kind of hack, could you supply a comment why this is important?